### ISSUE TITLE
[Feature Request] [HL2] Show on steam what custom workshop campaign is being played for friends.

### ISSUE THREAD

**Issue by Reddbr34** [ValveSoftware/Source-1-Games#6653]
Just like on the title, display on steam what the campaign is being played from the workshop, so people on the friend list can see what is being played, rather than just show Main Menu.
(for Half-life 2)

**Comment by Ethanol-001** [comment]
Agreed, or at least say anything other than "Main Menu". I don't want people to think I'm some sort of freak who just sits on the main menu all the time

**Comment by misyltoad** [comment]
Will be in the next update.
